app_dart/lib/src/service/branch_service.dart

```diff
@@ -113,4 +113,22 @@ class BranchService {
 
     throw InternalServerError('Failed to find a revision to branch Flutter recipes for $branch');
   }
+
+  Future<List<Map<String, String>>> getStableBetaDevBranches(
+      {required gh.GitHub github, required gh.RepositorySlug slug}) async {
+    final String devSha = (await github.repositories.getBranch(slug, 'dev')).commit!.sha!;
```

Comment:
Sorry for the confusion, but dev isn't actually a branch name. It refers to the latest google3 candidate branch. This is confusing because we've mostly deprecated it, but there's still a few references.

It should be the most recent of the flutter regexes

Reply:
Thanks for the clarification! Switched back to be the latest candidate branch based on version info in regex